src/core/audio/AudioPort.cpp

```diff
@@ -195,13 +195,13 @@ void AudioPort::doProcessing()
 		// has vol model only
 		else if( m_volumeModel )
 		{
-			ValueBuffer * volBuf = m_volumeModel->valueBuffer();
+			auto* volBuf = m_volumeModel->valueBuffer();
 
 			if( volBuf )
 			{
 				for( f_cnt_t f = 0; f < fpp; ++f )
 				{
-					float v = volBuf->values()[ f ] * 0.01f;
+					float v = volBuf->data()[ f ] * 0.01f;
```

Comment:
```suggestion
					float v = volBuf->data()[f] * 0.01f;
```

Reply:
Using the `data()` method feels kinda weird.

Personally I would just use:
```c++
(*volBuf)[f]
```

Reply:
Or would it be a good idea to use `at()` here

Reply:
If you're not sure whether `f` could be out of bounds, `at` would be good

Reply:
```cpp
(*volBuf)[f]
```

I used this for now.